src/Engine/ProtoCore/DSASM/Executable.cs

```diff
@@ -50,6 +50,9 @@ public enum OffsetConstants
         public ProtoCore.DSASM.ProcedureTable[] procedureTable { get; set; }
         public ProtoCore.DSASM.SymbolTable[] runtimeSymbols { get; set; }
 
+        public List<CodeBlock> CodeBlocks { get; set; }
```

Comment:
I kept it to prevent regression, but ill add notes to the other task to explore if 'CodeBlocks is still required
Ive added an entry in: http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-6193